bin/update_dependencies.py

```diff
@@ -12,21 +13,39 @@
 
 # CUSTOM_COMPILE_COMMAND is a pip-compile option that tells users how to
 # regenerate the constraints files
-os.environ['CUSTOM_COMPILE_COMMAND'] = "bin/update_constraints.py"
+os.environ['CUSTOM_COMPILE_COMMAND'] = "bin/update_dependencies.py"
 subprocess.check_call([
     'pip-compile',
     '--allow-unsafe',
     '--upgrade',
     'cibuildwheel/resources/constraints.in',
 ])
-for python_version in ['27', '35', '36', '37']:
-    subprocess.check_call([
-        f'./env{python_version}/bin/pip-compile',
-        '--allow-unsafe',
-        '--upgrade',
-        'cibuildwheel/resources/constraints.in',
-        '--output-file', f'cibuildwheel/resources/constraints-python{python_version}.txt'
-    ])
+
+if '--no-docker' in sys.argv:
+    for python_version in ['27', '35', '36', '37']:
```

Comment:
could the `['27', '35', '36', '37']` part be extracted to a common variable? `PYTHON_VERSIONS` or something?